test/functionalTest/cases/1444_err_msg_duplicate_attr/create_entity_dup_attr.test

```diff
@@ -31,6 +31,7 @@ brokerStart CB
 
 #
 # 01. POST /v2/entities, creating first entity with id E1
+# FIXME P10: obsolete preamble? steps need update?
```

Comment:
This whole test seems unfinished. Lots of mistakes ...
It is called
  GET Entity V2 Too Many Entities

So, I imagine the idea for it is to create two entities with the same ID but different types and then do a GET /v2/entities/{ID} and see an error.

Easy to fix.
The test in its current state is pretty useless.

For a new PR ?